When running "./configure --disable-shared --enable-
static" the build fails with the following error:

gcc -g -Wall -Wstrict-prototypes -Wimplicit-function-
declaration -g -O2 -o nagle_snd
nagle_snd.o ../../src/lib/.libs/libsctp.a ../../src/t
estlib/.libs/libsctputil.a
../../src/testlib/.libs/libsctputil.a(sctputil.o): In
function `test_peer_addr':
/home/roubert/src/lksctp/lksctp-tools-
1.0.6/src/testlib/sctputil.c:376: undefined reference
to `sctp_getpaddrs'
/home/roubert/src/lksctp/lksctp-tools-
1.0.6/src/testlib/sctputil.c:382: undefined reference
to `sctp_freepaddrs'
/home/roubert/src/lksctp/lksctp-tools-
1.0.6/src/testlib/sctputil.c:416: undefined reference
to `sctp_freepaddrs'
/home/roubert/src/lksctp/lksctp-tools-
1.0.6/src/testlib/sctputil.c:400: undefined reference
to `sctp_freepaddrs'
collect2: ld returned 1 exit status
make[3]: *** [nagle_snd] Error 1

Changing the order of libsctp.a and libsctputil.a
fixes this. The attached patch updates
src/apps/Makefile.am and src/func_tests/Makefile.am
to do that.

Patch submitted by Fredrik Roubert <roubert@sf.net>.
Patch needed reformatation because it did not applies as it was.

diff -Naur a/src/apps/Makefile.am b/src/apps/Makefile.am
--- a/src/apps/Makefile.am	2014-07-04 14:57:42.707977872 +0200
+++ b/src/apps/Makefile.am	2014-07-04 14:59:12.336231677 +0200
@@ -8,8 +8,8 @@
 INCLUDES = -I. -I$(top_srcdir)/src/include -I$(top_srcdir)/src/testlib 
 AM_CFLAGS = -g -Wall -Wstrict-prototypes -Wimplicit-function-declaration
 AM_LDFLAGS = 
-LDADD = $(top_builddir)/src/lib/libsctp.la \
-	$(top_builddir)/src/testlib/libsctputil.la
+LDADD = $(top_builddir)/src/testlib/libsctputil.la \
+	$(top_builddir)/src/lib/libsctp.la
 
 # programs to be installed with the distriubution
 bin_PROGRAMS = sctp_darn sctp_test
diff -Naur a/src/func_tests/Makefile.am b/src/func_tests/Makefile.am
--- a/src/func_tests/Makefile.am	2014-07-04 15:04:48.678188041 +0200
+++ b/src/func_tests/Makefile.am	2014-07-04 15:13:24.624769819 +0200
@@ -8,8 +8,8 @@
 INCLUDES = -I. -I$(top_srcdir)/src/include -I$(top_srcdir)/src/testlib 
 AM_CFLAGS = -g -Wall -Wstrict-prototypes -Wimplicit-function-declaration
 AM_LDFLAGS = -lpthread
-LDADD = $(top_builddir)/src/lib/libsctp.la \
-	$(top_builddir)/src/testlib/libsctputil.la
+LDADD = $(top_builddir)/src/testlib/libsctputil.la \
+	$(top_builddir)/src/lib/libsctp.la
 
 V6FLAGS = -DCONFIG_IPV6=1 -DTEST_V6=1 ${DEFS} ${INCLUDES} ${CFLAGS}
 
